src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -147,14 +147,15 @@
     <Compile Include="$(CommonTestPath)\System\Diagnostics\DebuggerAttributes.cs">
       <Link>Common\System\Diagnostics\DebuggerAttributes.cs</Link>
     </Compile>
-  </ItemGroup>
-  <!-- Performance Tests -->
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">
+    <!-- Performance Tests -->
     <Compile Include="Performance\Perf.HashTable.cs" />
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
   </ItemGroup>
+  <PropertyGroup Condition="'$(Performance)' == 'true'">
```

Comment:
Honestly I think we can just unconditionally include it. I had meant for it to be a kind of "flag" property meaning "Yes, this assembly has performance tests in it". I don't think there's any harm in including the assembly-level attributes, although I'm not sure how exactly they're interpreted.